Vagrantfile

```diff
@@ -46,6 +46,8 @@ Vagrant.configure(2) do |config|
     echo "Creating a virtualenv and installing requirements!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
     echo "================================================================================"
     mkvirtualenv alliance
+    workon alliance
+    cd /vagrant
```

Comment:
Correct. In order to do `pip install -r requirements.txt` we need to cd to the project directory where requirements.txt lives, which is `/vagrant` on the VM in our case.

The old version of the Vagrantfile didn't have this, but instead tried to make the vagrant user's home directory mirror the project directory. Unfortunately, that breaks the ability to ssh into the VM because  it changes the permissions on the directory.